rust/lance/src/index/vector/ivf/builder.rs

```diff
@@ -81,7 +92,79 @@ pub(super) async fn build_partitions(
     )
     .await?;
 
-    write_index_partitions(writer, ivf, Some(stream), None).await?;
+    write_pq_partitions(writer, ivf, Some(stream), None).await?;
 
     Ok(())
 }
+
+/// Build specific partitions of IVF index.
+///
+///
+#[allow(clippy::too_many_arguments)]
+#[instrument(level = "debug", skip(writer, auxiliary_writer, data, ivf, pq))]
```

Comment:
Maybe just do `skip_all` here?  I feel like this will log too much information.  Though I guess this method won't get called all that often.